vsphere/resource_vsphere_virtual_machine.go

```diff
@@ -283,6 +283,23 @@ func resourceVSphereVirtualMachine() *schema.Resource {
 			Computed:    true,
 			Description: "The power state of the virtual machine.",
 		},
+		"usb_controller": {
+			Type:        schema.TypeList,
+			Optional:    true,
+			Description: "A specification for a USB controller on the virtual machine.",
+			Elem: &schema.Resource{
+				Schema: map[string]*schema.Schema{
+					"usb_version": {
+						Type:         schema.TypeString,
+						Optional:     true,
+						Default:      "2.0",
+						Description:  "The version of the USB controller.",
+						ValidateFunc: validation.StringInSlice([]string{"2.0", "3.1"}, false),
```

Comment:
Maybe something like

```go
var usbVersions = []string{"2.0", "3.1"}
```
But account for the 1.x (more than one 1.x version?)

And then:

```go
ValidateFunc: validation.StringInSlice(usbVersions, false),
```